uaclient/contract.py

```diff
@@ -162,6 +163,29 @@ def request_machine_token_update(
             detach=False,
         )
 
+    def request_machine_activity(
+        self,
+        contract_id: str,
+        machine_id: str,
```

Comment:
I think we also need a machine_token in this method which needs to be placed in headers of the request 
headers.update({"Authorization": "Bearer {}".format(machine_token)})
We can test this against contracts.develop.canonical.com. even though staging doesn't have it.